CollapseLauncher/Classes/Helper/Animation/AnimationHelper.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Hi3Helper;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Hosting;
+using System;
+using System.Threading.Tasks;
+
+namespace CollapseLauncher.Helper.Animation
+{
+    internal static class AnimationHelper
+    {
+        internal static async Task StartAnimation(this UIElement element, TimeSpan duration, params KeyFrameAnimation[] animBase)
+        {
+            foreach (KeyFrameAnimation anim in animBase)
+            {
+                if (element.DispatcherQueue.HasThreadAccess)
+                {
+                    anim.Duration = duration;
+                    element.StartAnimation(anim);
+                }
+                else
+                    element.DispatcherQueue.TryEnqueue(() =>
+                    {
+                        anim.Duration = duration;
+                        element.StartAnimation(anim);
+                    });
+            }
+            await Task.Delay(duration);
+            foreach (var item in animBase)
+            {
+                if (element.DispatcherQueue.HasThreadAccess)
+                    element.StopAnimation(item);
+                else
+                    element.DispatcherQueue.TryEnqueue(() =>
+                        element.StopAnimation(item));
+            }
+        }
+
+        internal static void EnableImplicitAnimation(this UIElement element, bool recursiveAssignment = false, CompositionEasingFunction easingFunction = null)
+        {
+            try
+            {
+                Visual rootFrameVisual = ElementCompositionPreview.GetElementVisual(element);
+                Compositor compositor = rootFrameVisual.Compositor;
+
+                ImplicitAnimationCollection animationCollection = rootFrameVisual.ImplicitAnimations != null ? rootFrameVisual.ImplicitAnimations : compositor.CreateImplicitAnimationCollection();
```

Comment:
Just for the sake of code cleanliness, maybe spread this on 2 lines if possible

Reply:
Adjusted in 334d0516f0433263f343f8743aee4fa005de92ec